File: selector/src/main/java/com/luck/picture/lib/interfaces/OnRecyclerViewScrollStateListener.java
package com.luck.picture.lib.interfaces;

/**
 * @author：luck
 * @date：2020-04-14 18:44
 * @describe：OnRecyclerViewScrollStateListener
 */
public interface OnRecyclerViewScrollStateListener {

    /**
     * RecyclerView Scroll Fast
     */
    void onScrollFast();

    /**
     * RecyclerView Scroll Slow
     */
    void onScrollSlow();
}

File: selector/src/main/java/com/luck/picture/lib/interfaces/OnRecyclerViewScrollStateListener.kt
package com.luck.picture.lib.interfaces

/**
 * @author：luck
 * @date：2020-04-14 18:44
 * @describe：OnRecyclerViewScrollStateListener
 */
interface OnRecyclerViewScrollStateListener {
    /**
     * RecyclerView Scroll Fast
     */
    fun onScrollFast()

    /**
     * RecyclerView Scroll Slow
     */
    fun onScrollSlow()
}